cmd/pyroscope/help-all.txt.tmpl

```diff
@@ -406,7 +530,7 @@ Usage of ./pyroscope:
   -querier.max-query-parallelism int
     	Maximum number of queries that will be scheduled in parallel by the frontend.
   -querier.query-store-after duration
-    	The time after which a metric should be queried from storage and not just ingesters. 0 means all queries are sent to store. If this option is enabled, the time range of the query sent to the store-gateway will be manipulated to ensure the query end is not more recent than 'now - query-store-after'. (default 4h0m0s)
+    	The time after which a metric should be queried from storage and not just ingesters. 0 means all queries are sent to store. If this option is enabled, the time range of the query sent to the store-gateway will be manipulated to ensure the query end is not more recent than 'now - query-store-after'. (default 12h0m0s)
```

Comment:
Can you explain this change?

Reply:
Not sure it's intentional, I would actually prefer 4h for the future.